sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1966,9 +2045,15 @@ pi_result piextQueueGetNativeHandle(pi_queue Queue,
 pi_result piextQueueCreateWithNativeHandle(pi_native_handle NativeHandle,
                                            pi_context Context,
                                            pi_queue *Queue) {
-  assert(NativeHandle);
-  assert(Context);
-  assert(Queue);
+  if (!NativeHandle) {
+    return PI_INVALID_VALUE;
+  }
+  if (Context == nullptr) {
+    return PI_INVALID_CONTEXT;
+  }
+  if (Queue == nullptr) {
+    return PI_INVALID_QUEUE;
+  }
```

Comment:
Can we please make it more concise by using a macro like ZE_CALL?
E.g. 

PI_ASSERT(NativeHandle, PI_INVALID_VALUE)
PI_ASSERT(Context, PI_INVALID_CONTEXT)
PI_ASSERT(Queue, PI_INVALID_Queue)

Reply:
good idea. will update.